aspnetcore/security/authentication/accconfirm.md

```diff
@@ -86,128 +74,113 @@ You generally want to prevent new users from posting any data to your web site b
 
-   * Create a class to fetch the secure email key. For this sample, the `AuthMessageSenderOptions` class is created in the *Services/AuthMessageSenderOptions.cs* file.
+Create a class to fetch the secure email key. For this sample, the `AuthMessageSenderOptions` class is created in the *Services/AuthMessageSenderOptions.cs* file.
 
-[!code-csharp[Main](accconfirm/sample/WebApplication3/src/WebApplication3/Services/AuthMessageSenderOptions.cs?range=8-12)]
+[!code-csharp[Main](accconfirm/sample/WebApp1/Services/AuthMessageSenderOptions.cs?name=snippet1)]
 
 Set the `SendGridUser` and `SendGridKey` with the [secret-manager tool](../app-secrets.md). For example:
 
 ```none
-C:\WebApplication3\src\WebApplication3>dotnet user-secrets set SendGridUser RickAndMSFT
+C:\WebAppl\src\WebApp1>dotnet user-secrets set SendGridUser RickAndMSFT
 info: Successfully saved SendGridUser = RickAndMSFT to the secret store.
 ```
 
-On Windows, Secret Manager stores your keys/value pairs in a *secrets.json* file in the %APPDATA%/Microsoft/UserSecrets/<**userSecretsId**> directory. The **userSecretsId** directory can be found in your *project.json* file. For this example, the first few lines of the *project.json* file are shown below:
+On Windows, Secret Manager stores your keys/value pairs in a *secrets.json* file in the %APPDATA%/Microsoft/UserSecrets/<WebAppName-userSecretsId> directory.
 
-[!code-json[Main](../../security/authentication/accconfirm/sample/WebApplication3/src/WebApplication3/project.json?highlight=3&range=1-6)]
+The contents of the *secrets.json* file are not encrypted. The *secrets.json* file is shown below (the `SendGridKey` value has been removed.)
 
-At this time, the contents of the *secrets.json* file are not encrypted. The *secrets.json* file is shown below (the sensitive keys have been removed.)
-
-```json
-{
-  "SendGridUser": "RickAndMSFT",
-  "SendGridKey": "",
-  "Authentication:Facebook:AppId": "",
-  "Authentication:Facebook:AppSecret": ""
-}
-```
+  ```json
+  {
+    "SendGridUser": "RickAndMSFT",
+    "SendGridKey": "<key removed>"
+  }
+  ```
 
 ### Configure startup to use `AuthMessageSenderOptions`
 
 Add the dependency `Microsoft.Extensions.Options.ConfigurationExtensions` in the project.json file.
 
 Add `AuthMessageSenderOptions` to the service container at the end of the `ConfigureServices` method in the *Startup.cs* file:
 
-[!code-csharp[Main](../../security/authentication/accconfirm/sample/WebApplication3/src/WebApplication3/Startup.cs?highlight=4&range=62-65)]
+[!code-csharp[Main](accconfirm/sample/WebApp1/Startup.cs?name=snippet1&highlight=26)]
 
-### Configure the `AuthMessageSender` class
+### Configure the AuthMessageSender class
 
 This tutorial shows how to add email notification through [SendGrid](https://sendgrid.com/), but you can send email using SMTP and other mechanisms.
 
 * Install the SendGrid.NetCore NuGet package. From the Package Manager Console,  enter the following the following command:
```

Comment:
* Format package name
* The name here is out of sync with the following command line example